suite-native/labeling/src/components/ExportBip329InfoOverlay.tsx

```diff
@@ -0,0 +1,22 @@
+import { Box, PictogramTitleHeader } from '@suite-native/atoms';
+import { Translation } from '@suite-native/intl';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+const overlayStyle = prepareNativeStyle(utils => ({
+    justifyContent: 'center',
+    alignItems: 'center',
+    padding: utils.spacings.sp16,
+}));
+
+export const ExportBip329InfoOverlay = () => {
```

Comment:
It's not overlay, what about different name like this:
```suggestion
export const ExportBip329InfoHeader = () => {
```

Reply:
83683cee0b5927f74c61854e9f40c18f014a8b99